cypress/e2e/tests/migration/applicationinventory/manageimports/csv.test.ts

```diff
@@ -16,20 +16,18 @@ limitations under the License.
 /// <reference types="cypress" />
 
 import { cleanupDownloads, login, openManageImportsPage } from "../../../../../utils/utils";
-import { Application } from "../../../../models/migration/applicationinventory/application";
 import { kebabMenuItem } from "../../../../views/applicationinventory.view";
 import { manageImportsActionsButton } from "../../../../views/common.view";
 
 describe(["@tier3"], "Manage imports tests", function () {
     before("Login", function () {
         login();
         cy.visit("/");
-        Application.open();
-        openManageImportsPage();
     });
 
     it("Download CSV template", function () {
-        cy.get(manageImportsActionsButton).eq(0).click({ force: true });
+        openManageImportsPage();
+        cy.get(manageImportsActionsButton).click();
```

Comment:
Is this needed? You are already clicking on that button inside the `openManageImportsPage` method

Reply:
Thanks @abrugaro !

Yes, it's needed. We are clicking the kebab menu on the 'Application imports' page .
whereas we are clicking the top kebab menu on the 'Application inventory' page' through  openManageImportsPage() .
(openManageImportsPage()  in turn calls application_inventory_kebab_menu() where the top kebab menu on the 'Application inventory' is being clicked.